.stickler.yml

```diff
@@ -0,0 +1,11 @@
+linters:
+    flake8:
+    eslint:
+        config: './eslint.config'
+  
+
+files:
+    ignore:
+        - 'src/generated/*'
+        - 'webroot/bower_components/*'
+        - 'js/index.js'
```

Comment:
Let's use the content here instead: https://github.com/microverseinc/linters-config/blob/master/react-redux/.stickler.yml